cwms-data-api/src/main/java/cwms/cda/ApiServlet.java

```diff
@@ -149,31 +161,33 @@
  *
  */
 @WebServlet(urlPatterns = { "/catalog/*",
-        "/auth/*",
-        "/swagger-docs",
-        "/timeseries/*",
-        "/offices/*",
-        "/states/*",
-        "/counties/*",
-        "/location/*",
-        "/locations/*",
-        "/parameters/*",
-        "/timezones/*",
-        "/units/*",
-        "/ratings/*",
-        "/levels/*",
-        "/basins/*",
-        "/blobs/*",
-        "/clobs/*",
-        "/pools/*",
-        "/specified-levels/*",
-        "/forecast-spec/*",
-        "/forecast-instance/*",
-        "/standard-text-id/*",
-        "/projects/*",
-        "/properties/*",
-        "/lookup-types/*",
-        "/embankments/*",
+    "/auth/*",
+    "/swagger-docs",
+    "/timeseries/*",
+    "/offices/*",
+    "/states/*",
+    "/counties/*",
+    "/location/*",
+    "/locations/*",
+    "/parameters/*",
+    "/timezones/*",
+    "/units/*",
+    "/ratings/*",
+    "/levels/*",
+    "/basins/*",
+    "/blobs/*",
+    "/clobs/*",
+    "/pools/*",
+    "/specified-levels/*",
+    "/forecast-spec/*",
+    "/forecast-instance/*",
+    "/standard-text-id/*",
+    "/projects/*",
+    "/project-locks/*",
+    "/project-lock-rights/*",
+    "/properties/*",
+    "/lookup-types/*",
+    "/embankments/*",
         "/turbines/*"
```

Comment:
You know, maybe we should just change this to * and have javalin also sort out the static files, this thing is getting a smidge long.

Reply:
It doesn't bother me but if we're going to do that lets make it a standalone pr.

Reply:
Agreed. Was mostly just commenting on just how long that list is getting.